src/main/kotlin/org/taktik/icure/asyncdao/impl/MaintenanceTaskDAOImpl.kt

```diff
@@ -55,13 +55,18 @@ class MaintenanceTaskDAOImpl(
 	@View(name = "by_date", map = "function(doc) { if (doc.java_type === 'org.taktik.icure.entities.MaintenanceTask' && !doc.deleted) emit(doc.created)}")
 	override fun listMaintenanceTasksAfterDate(date: Long) = flow {
 		val client = couchDbDispatcher.getClient(dbInstanceUrl)
-		emitAll(client.queryView<Long, Void>(createQuery(client, "by_date").startKey(999999999999L).endKey(date).descending(true).includeDocs(false)).map { it.id })
+		emitAll(client.queryView<Long, Void>(createQuery(client, "by_date").endKey(date).descending(true).includeDocs(false)).map { it.id })
```

Comment:
I know we had to add the startKey in order to include all values properly corresponding to the filter. Otherwise, they are not retrieved by the view

Reply:
Added the test in the latest commit